docs/UserGuide.adoc

```diff
@@ -540,54 +540,53 @@ Shows the 2nd version of offline copy of bookmark 1. If it is annotated,
 then annotations are shown as well.
 
 
-=== Adding reminder: `reminder`
+=== Adding a reminder: `reminder`
 
-Adds a reminder to visit a bookmark. You can set the time for the reminder. The app
-will inform you at that time. You can also choose whether to open the bookmark.
+Adds a reminder to visit a bookmark. You can set the time for the reminder.
+You can also choose when to open the bookmark.
 The default setting is open.
```

Comment:
Can I check, is `the time for the reminder` (line 545) the same as `when to open the bookmark` (line 546)? It looks like you can set two different times. Also, suggested change below:
```suggestion
```

Reply:
Fixed. Can you check again the description and the example?